scene/gui/color_picker_shape.cpp

```diff
@@ -762,8 +765,12 @@ void ColorPickerShapeCircle::update_circle_cursor(const Vector2 &p_color_change_
 }
 
 void ColorPickerShapeCircle::_initialize_controls() {
+	circle_hb = memnew(HBoxContainer);
+	circle_hb->set_h_size_flags(Control::SIZE_EXPAND | Control::SIZE_SHRINK_CENTER);
+	color_picker->shape_container->add_child(circle_hb);
```

Comment:
Making `shape_container` center-aligned has the same effect, though not sure if it doesn't break anything.

Reply:
It works, changes went down from +11 to +3, Thank You!